nautobot_golden_config/models.py

```diff
@@ -368,6 +368,8 @@ def to_objectchange(self, action, *, related_object=None, object_data_extra=None
             related_object=related_object,
         )
 
+    is_dynamic_group_associable_model = False
```

Comment:
Maybe it'd be correct to flag all of golden-config's data models thusly?

Reply:
I'm happy to make that change if it's ok with @itdependsnetworks and @jeffkala

Reply:
What would the advantage be?